Sources/Public/Animation/LottiePlaybackMode.swift

```diff
@@ -7,6 +7,13 @@ import Foundation
 
 /// Configuration for how a Lottie animation should be played
 public enum LottiePlaybackMode: Hashable {
+
+  /// The position within a marker.
+  public enum MarkerPosition: Hashable {
```

Comment:
Can this be a top-level `LottieMarkerPosition` type below the `LottiePlaybackMode` declaration?